Source/SelfService/Web/stores/microservice.ts

```diff
@@ -132,7 +132,7 @@ export async function deleteMicroservice(
     return response;
 }
 
-const saveMicroservice = async (kind: string, input: any): Promise<boolean> => {
+const saveMicroservice = async (kind: string, input: any): Promise<any> => {
```

Comment:
If we don't use the actual data for anything, this should still be a boolean.

It feels a bit confusing if it returns either the data for a microservice, or false.

Reply:
Changed it back to boolean, any potential errors are still handled through exceptions so that it bubbles up all the way to the frontend and snackbar. In the case of it returning `false` I made a basic snackbar error to appear.